vpr/src/base/read_options.cpp

```diff
@@ -1791,6 +1826,14 @@ argparse::ArgumentParser create_arg_parser(const std::string& prog_name, t_optio
         .default_value("appack")
         .show_in(argparse::ShowIn::HELP_ONLY);
 
+    ap_grp.add_argument<e_ap_detailed_placer, ParseAPDetailedPlacer>(args.ap_detailed_placer, "--ap_detailed_placer")
+        .help(
+            "Controls which Detailed Placer to use in the AP Flow.\n"
+            " * none: Do not perform any detailed placement. Directly routes the result of the Full Legalizer.\n"
```

Comment:
Delete "Directly routes the result ..." (Implies --route is on, which may not be the case."
Instead could say "i.e. the output of the analytic placer will be used without modification."